docs/user-guide/release-notes/2.7.x.rst

```diff
@@ -46,6 +69,13 @@ Known Issues
 Client Changes
 --------------
 
+* Admin and consumer Pulp clients now support -v and -vv flags to get additional
+  information in case of failures. Exceptions raised for CLI and API level
```

Comment:
-v and -vv flags display additional info not only in case of failure, it just displays INFO and DEBUG mode depending on verbosity